### ISSUE TITLE
The model parameters in the code are inconsistent with those in the  pretrained model

### ISSUE THREAD

**Issue by neiheih** [yhlleo/DeepSegmentor#4]
Error(s) in loading state_dict for RoadNet
For roadnet I have download the pretrained model form https://pan.baidu.com/s/1GEevUwp1xrEtASjixawrwg
parameters of the model defied  in the code:
![my](https://user-images.githubusercontent.com/53416220/68388569-eb739280-019b-11ea-896b-d5a0728231f3.PNG)
parameters in the pretrained model:
![yours](https://user-images.githubusercontent.com/53416220/68388609-0ba35180-019c-11ea-8f05-e55c95b8a3be.PNG)
the model in the code seems missing some 1*1 conv layers.
May you have a look at this? thanks!

**Comment by yhlleo** [comment]
Sorry, does the training works?  I tested to load the saved models:

 - Google Drive file:
```
...
----------------- End -------------------
dataset [RoadNetDataset] was created
initialize network with xavier
model [RoadNetModel] was created
loading the model from ./checkpoints/roadnet/latest_net_G.pth
---------- Networks initialized -------------
[Network G] Total number of parameters : 17.075 M
-----------------------------------------------
...
```

 - Baidu Yun
```
...
----------------- End -------------------
dataset [RoadNetDataset] was created
initialize network with xavier
model [RoadNetModel] was created
loading the model from ./checkpoints/roadnet/latest_net_G.pth
---------- Networks initialized -------------
[Network G] Total number of parameters : 17.075 M
-----------------------------------------------
...
```

It seems there are no problems in the saved model and defined model.

**Comment by neiheih** [comment]
Thanks, I just used parts of trainning code, let me have a check.

**Comment by neiheih** [comment]
I have run your code, and a error occuerd when calling func  __patch_instance_norm_state_dict.
![1](https://user-images.githubusercontent.com/53416220/68459497-8cb42480-0240-11ea-9a18-b9070a795e55.PNG)
![2](https://user-images.githubusercontent.com/53416220/68459517-98075000-0240-11ea-9a16-807670cc504e.PNG)
Is it necessary before loading state_dict?
When I skipped  it, another error occuerd.
![3](https://user-images.githubusercontent.com/53416220/68459523-9ccc0400-0240-11ea-80b2-8a4dac5efc4f.PNG)

**Comment by yhlleo** [comment]
It could be a problem of mismatching version. I have provided the configuration file [environment.yml](https://github.com/yhlleo/DeepSegmentor/blob/master/environment.yml). Please check the PyTorch version.

**Comment by neiheih** [comment]
I am using pytorch vision of 1.3.0. Any advice to let the model run on this version?

**Comment by yhlleo** [comment]
Sorry, I don't carefully compare the differences between the two versions. If you want to use the architecture or loss terms, you can use them in the form of your version.
